src/Tokenizer.js

```diff
@@ -67,24 +67,24 @@ export function slice (begin, end) {
  */
 export function token (type) {
 	switch (type) {
-		// * / comment tokens
+		// * / comment token
 		case 42: case 47:
```

Comment:
What test fails when you remove the `*(42)`? andd is it just a whitespace diff?

Reply:
basically each test containing a block comment, because Parser peeks next character when it meets `/`, it doesnt know if it should start consuming a comment or not unless it makes sure that the next character is `/` or `*`, so removing `*` from here makes Parser not consuming block comments at all